client/my-sites/plans/current-plan/jetpack-checklist/footer.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+import { localize } from 'i18n-calypso';
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSite } from 'state/ui/selectors';
+import { recordTracksEvent } from 'state/analytics/actions';
+import { untrailingslashit } from 'lib/route';
+import Button from 'components/button';
+import Card from 'components/card';
+
+class JetpackChecklistFooter extends PureComponent {
+	handleWPAdminLink = () => {
+		this.props.recordTracksEvent( 'calypso_checklist_wpadmin' );
+	};
+
+	render() {
+		const { translate, wpAdminUrl } = this.props;
+		return (
+			<Card compact className="jetpack-checklist__footer">
+				<p>{ translate( 'Return to your self-hosted WordPress dashboard.' ) }</p>
+				<Button compact href={ wpAdminUrl } onClick={ this.handleWPAdminLink() }>
```

Comment:
I don't think this should be invoked here:

```suggestion
				<Button compact href={ wpAdminUrl } onClick={ this.handleWPAdminLink }>
```

Reply:
Good catch!